### PR TITLE
:clock2: Reflect minutes passed in the hour hand

### PR DESCRIPTION
**Pull request by regexb**

### CODE REVIEW

**Comment by samccone** [comment]
can you check off the todo in the readme :)

**Comment by regexb** [comment]
Readme updated

File: analog-clock.html [hunk 1]
```diff
@@ -73,9 +73,10 @@
     tick: function() {
       var self = this;
       var d = new Date();
+      var minutes = d.getMinutes() / 60
```
Comment (samccone) [inline analog-clock.html:76]: this is not quite minutes right :) it is a fraction of 60 :) not sure the best name for it ha

Comment (samccone) [inline analog-clock.html:76]: does not really matter so much :ship:

**Comment by samccone** [comment]
thanks :clap:

**Pull request merged**